### PR TITLE
[10.x] update doc blocks in MySqlSchemaState

### PR DESCRIPTION
**Pull request by saMahmoudzadeh**
This pull request aims to enhance the documentation blocks within the MySqlSchemaState class.

### CODE REVIEW

File: src/Illuminate/Database/Schema/MySqlSchemaState.php [hunk 1]
```diff
@@ -15,6 +15,9 @@ class MySqlSchemaState extends SchemaState
      * @param  \Illuminate\Database\Connection  $connection
      * @param  string  $path
      * @return void
+     *
+     * @throws \Illuminate\Contracts\Filesystem\FileNotFoundException
+     * @throws Exception
```
Comment (crynobone) [inline src/Illuminate/Database/Schema/MySqlSchemaState.php:20]:
```suggestion
     * @throws \Exception
```

File: src/Illuminate/Database/Schema/MySqlSchemaState.php [hunk 2]
```diff
@@ -49,6 +54,8 @@ protected function removeAutoIncrementingState(string $path)
      *
      * @param  string  $path
      * @return void
+     *
+     * @throws Exception
```
Comment (crynobone) [inline src/Illuminate/Database/Schema/MySqlSchemaState.php:58]:
```suggestion
     * @throws \Exception
```

File: src/Illuminate/Database/Schema/MySqlSchemaState.php [hunk 3]
```diff
@@ -144,6 +151,8 @@ protected function baseVariables(array $config)
      * @param  callable  $output
      * @param  array  $variables
      * @return \Symfony\Component\Process\Process
+     *
+     * @throws Exception
```
Comment (crynobone) [inline src/Illuminate/Database/Schema/MySqlSchemaState.php:155]:
```suggestion
     * @throws \Exception
```

File: src/Illuminate/Database/Schema/MySqlSchemaState.php (hunk 2 above)
Comment (saMahmoudzadeh) [reply src/Illuminate/Database/Schema/MySqlSchemaState.php]: Thanks

**Comment by saMahmoudzadeh** [comment]
I have implemented your suggestions. Thanks!

@crynobone

File: src/Illuminate/Database/Schema/MySqlSchemaState.php [hunk 4]
```diff
@@ -2,7 +2,6 @@
 
 namespace Illuminate\Database\Schema;
 
-use Exception;
```
Comment (driesvints) [inline src/Illuminate/Database/Schema/MySqlSchemaState.php:5]: Please re-add this as the type is still used in `executeDumpProcess`

**Review by driesvints — CHANGES_REQUESTED** [review]

File: src/Illuminate/Database/Schema/MySqlSchemaState.php (hunk 4 above)
Comment (saMahmoudzadeh) [reply src/Illuminate/Database/Schema/MySqlSchemaState.php]: Thank you! It was applied.

**Comment by driesvints** [comment]
Actually I'm sorry but these methods don't throw these exceptions directly. We usually don't add docblocks for methods that don't throw exceptions directly, only in select cases. Thanks though!

**Pull request closed without merging**